src/todo/helper.lua

```diff
@@ -13,20 +13,23 @@ function todo.log(message)
 end
 
 function todo.get_player_list()
-    local result = {{"todo.unassigned"} }
+    local result = { {"todo.unassigned"} }
 
     for _, player in pairs(game.players) do
         table.insert(result, player.name)
     end
 
     local lookup = {}
+    local count = 0
```

Comment:
You could fix the off by one error by doing the counting in the loop above. Then you get the real result.

Reply:
I have to trust you on that one. I'll check this in game tonight if I catch the time to. Why is it different in the two loops?